### DISCUSSION TITLE
AI  Wasserzähler  Port 3000

### DISCUSSION THREAD

**Discussion by andyunt** [jomjol/AI-on-the-edge-device#2580] [General]
In the old Version  i can read data with http://server-ip:3000/wasserzaehler.html
Is it possible to check the data in the new Version in the same way ?

I need the data in my Loxone- Miniserver.

Thanks

**Comment by ghost** [comment]
Did you looked at https://jomjol.github.io/AI-on-the-edge-device-docs/REST-API/ ?
